<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldLayerWidget</class>
 <widget class="QWidget" name="WorldLayerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>292</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="diffuselayout_5" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(67, 67, 67);</string>
     </property>
     <widget class="QWidget" name="diffuselayout_6" native="true">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>701</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(85, 85, 85);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>1327</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>World Properties</string>
      </property>
     </widget>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QWidget" name="diffuselayout_3" native="true">
     <property name="minimumSize">
      <size>
       <width>1327</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color: rgb(85, 85, 85);
</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>580</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>181</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="fogEnabled">
              <property name="text">
               <string>Enable Fog</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Fog Color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ColorPickerWidget" name="fogColorPicker" native="true">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>25</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(65, 65, 65);
border-radius:2px;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Near Distance</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="LCDSlider" name="fogStart" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>End Distance</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="LCDSlider" name="fogEnd" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>181</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="gridCheck">
              <property name="text">
               <string> Show Grid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Background Color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ColorPickerWidget" name="skyBg_2" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(65, 65, 65);
border-radius:2px;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>181</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>181</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="chooseSkyButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
background-color: rgb(71, 71, 71);
border-radius:2px;
border: 1px solid  rgb(223, 223, 223);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
background-color: rgb(223, 223, 223);
border: 2px solid  rgb(223, 223, 223);
color: rgb(115, 115, 115);
}
</string>
              </property>
              <property name="text">
               <string>Choose</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Sky</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="skyLabel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(99, 99, 99);
border-radius:2px;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearSkyButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>30</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>30</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
background-color: rgb(175, 175, 175);
border-style: inset;
border-radius: 2px;
text-decoration: none;
color: rgb(0, 0, 0);
}
QPushButton:hover
{
background-color: rgb(235, 235, 235);
border-style: inset;
}
</string>
              </property>
              <property name="text">
               <string>Clear</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="skyList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>181</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>900</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget#skyList {
border: 1px solid rgb(54, 54, 54);
background-color: rgb(74, 74, 74);
}
/*--------------------  list view scrollbar style  ------------------------------*/

 QScrollBar:vertical {
	 background-color: rgb(180, 180, 180);
     width: 15px;
     margin: 22px 0 22px 0;
 }

 QScrollBar::handle:vertical {
	background-color: rgb(97, 97, 97);
     min-height: 20px;
	 border-radius: 5px;
	 margin-left: 2px;
	 margin-right: 2px;
 }

 QScrollBar::handle:vertical:hover {
	background-color: rgb(110, 110, 110);
     min-height: 20px;
	 border-radius: 5px;
	 margin-left: 2px;
	 margin-right: 2px;
 }

 QScrollBar::add-line:vertical { 
	 background-color: rgb(180, 180, 180);
     height: 30px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical { 
	 background-color: rgb(180, 180, 180);
     height: 30px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }

 QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
     background: none;
 }

QScrollBar::up-arrow:vertical
{
    image: url(:/icons/up-arrow.svg);
}
QScrollBar::up-arrow:vertical:hover
{
	background-color: rgb(162, 162, 162);
    image: url(:/icons/up-arrow.svg);
}

QScrollBar::down-arrow:vertical
{
    image: url(:/icons/down-arrow.svg);
}

QScrollBar::down-arrow:vertical:hover
{
	background-color: rgb(162, 162, 162);
    image: url(:/icons/down-arrow.svg);
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPickerWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/colorpickerwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LCDSlider</class>
   <extends>QWidget</extends>
   <header>src/widgets/lcdslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
